Gaijinizer patch/Map215.txt

```diff
@@ -1919,19 +1968,19 @@ Yo, “Yotori” ......! ? What about a bird demon
 
 基本的に、こいつらは人間を『つがい』にする
 > CONTEXT: Map215/events/7/pages/0/48/Dialogue/other1
-Yes Yes. It is classified as Harpy or Saleen.
-Basically, they make humans "paired"#MTLed
+That's right. They're classified as a kind of
+harpy or siren, but all you need to know is
+their tendency to pair bond with humans.#Ron
 > END STRING
```

Comment:
continuing from the previous change:
>_**That's right. They're classified as a kind of
harpy, but all you need to know is their tendency
to pair bond with humans.**_